img/j/jetbrains.com.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 87.1 87.1"><path fill="#010101" d="M87 0H0v87h87z"/><path fill="#fff" d="M42.2 70.8H9.5v5.4h32.7Zm-34-48.3 2.3-2.3c.7.8 1.3 1.2 2.2 1.2.9 0 1.5-.6 1.5-1.8V11H18v8.6c0 1.7-.5 3-1.4 3.9a5 5 0 0 1-3.7 1.3c-2.4 0-3.8-1-4.8-2.3zm11.4-11.6h11v3.2h-7.2v2h6.5v3h-6.5v2.2h7.3v3.2h-11V10.9ZM36 14.2h-4V11h11.8v3.3h-4v10.3H36V14.2ZM9.6 30h6.5c1.6 0 2.9.5 3.7 1.2.5.6.8 1.3.8 2.2v.1c0 1.6-1 2.5-2 3 1.5.6 2.7 1.6 2.7 3.4 0 2.4-2 3.8-5.2 3.8H9.7V30Zm7.3 4.2c0-.8-.6-1.2-1.7-1.2h-1.9v2.4h1.8c1.1 0 1.8-.4 1.8-1.2zM15.6 38h-2.3v2.5h2.3c1.2 0 1.8-.5 1.8-1.2v-.1c0-.8-.5-1.2-1.8-1.2zm6.8-8.1h6c2.1 0 3.6.5 4.5 1.4.8.8 1.2 1.8 1.2 3.2 0 2.2-1 3.5-2.8 4.3l3.3 4.7h-4.4l-2.7-4h-1.2v4h-3.8V30zm6 6.5c1.2 0 2-.7 2-1.6 0-1.2-.8-1.7-2-1.7h-2.1v3.3z"/><path fill="#fff" d="M39.3 29.8H43l5.7 13.7h-4l-1-2.5h-5.2l-1 2.5h-3.9zm3.3 8.3-1.5-3.8-1.5 3.8zm10.5-8.3h-3.8v13.6h3.8zm1.4 0H58l5 7.2v-7.2h3.7v13.6h-3.3L58.3 36v7.4h-3.8Zm12.8 11.6 2.1-2.5c1.4 1 2.9 1.6 4.4 1.6 1 0 1.6-.3 1.6-1 0-.6-.5-.8-2.3-1.3-3-.6-5.2-1.5-5.2-4.2 0-2.6 2-4.4 5.3-4.4 2.3 0 4 .6 5.6 1.8l-2 2.7a6.6 6.6 0 0 0-3.7-1.4c-1 0-1.4.4-1.4.8v.1c0 .7.4 1 2.3 1.3 3.1.7 5.1 1.7 5.1 4.2 0 2.9-2.2 4.5-5.5 4.5a9 9 0 0 1-6.3-2.2z"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" style="background:#010101;fill:#fff" viewBox="0 0 87 87"><path d="M42.2 70.8H9.5v5.4h32.7ZM10.5 20.2c.7.8 1.3 1.2 2.2 1.2.9 0 1.5-.6 1.5-1.8V11H18v8.6c0 1.7-.5 3-1.4 3.9a5 5 0 0 1-3.7 1.3 6.3 6.3 0 0 1-4.7-2.3zm9.1-9.3h11v3.2h-7.2v2h6.5v3h-6.5v2.2h7.3v3.2h-11V10.9ZM36 14.2h-4V11h11.8v3.3h-4v10.3H36ZM9.6 30h6.5c2.6.1 4.5 1.3 4.5 3.4 0 1.3-1 2.6-2 3.1 1.5.6 2.7 1.6 2.7 3.4 0 2.4-2 3.8-5.2 3.8H9.7V30Zm7.3 4.2c0-.8-.6-1.2-1.7-1.2h-1.9v2.4h1.8c1.1 0 1.8-.4 1.8-1.2zM15.6 38h-2.3v2.5h2.3c1.2 0 1.8-.5 1.8-1.2v-.1c0-.8-.5-1.2-1.8-1.2Zm6.8-8.1h6c2.1 0 3.6.5 4.5 1.4.8.8 1.2 1.8 1.2 3.2 0 2.2-1 3.5-2.8 4.3l3.3 4.7h-4.4l-2.7-4h-1.2v4h-3.8V30Zm6 6.5c1.2 0 2-.7 2-1.6 0-1.2-.8-1.7-2-1.7h-2.1v3.3zm10.9-6.6H43l5.7 13.7h-4l-1-2.5h-5.2l-1 2.5h-3.9zm3.3 8.3-1.5-3.8-1.5 3.8zm10.5-8.3h-3.8v13.6h3.8zm1.4 0H58l5 7.2v-7.2h3.7v13.6h-3.3L58.3 36v7.4h-3.8zm12.8 11.6 2.1-2.5c1.4 1 2.9 1.6 4.4 1.6 1 0 1.6-.3 1.6-1 0-.6-.5-.8-2.3-1.3-3-.6-5.2-1.5-5.2-4.2 0-2.6 2-4.4 5.3-4.4 2.3 0 4 .6 5.6 1.8l-2 2.7a7 7 0 0 0-3.7-1.4c-1 0-1.4.3-1.4 1 0 .5.4.9 2.3 1.2 3.1.7 5.1 1.7 5.1 4.2 0 2.9-2.2 4.5-5.5 4.5a9 9 0 0 1-6.3-2.2z"/></svg>
```

Reply:
Is the background displayed correctly for you outside of a web browser? It's transparent for me, and while editing this is just confusing. Saving those couple of bytes is not worth the effort, I think.

Reply:
It is valid SVG 2 code, so it should work, but I guess your editor might not support it yet :shrug:

Declaring what background color the image should use seems more intuitive than declaring a path at 87,0;0,87;87,0 and filling it with the background color.

Reply:
That's all well and good, but really doesn't help me much. The editors available to me (under Linux) don't support it, and neither does https://editsvg.online/ - even worse, it's black-on-white there. (WTF? So much for standards...)

For the handful of logos that might benefit from this CSS style marking, it's not worth the hassle in my view. Furthermore, I'd like to keep manual editing to a minimum anyway (i.e. ideally just cropping and SVGOMG optimizing)